modules/tournament/src/main/Schedule.scala

```diff
@@ -72,8 +72,8 @@ case class Schedule(
             case _                            => s"${freq.toString} ${speed.trans}"
         case (Some(_), _) if full   => eliteXArena.txt(speed.trans)
         case (Some(_), _)           => eliteX.txt(speed.trans)
-        case (_, Some(max)) if full => s"≤${max.rating} ${xArena.txt(speed.trans)}"
-        case (_, Some(max))         => s"≤${max.rating} ${speed.trans}"
+        case (_, Some(max)) if full => maxRatingXArena.txt(max.rating, speed.trans)
+        case (_, Some(max))         => maxRatingX.txt(max.rating, speed.trans)
```

Comment:
there's literally no text to translate there. And it's only confusing to the translators: `≤ %1$s %2$s`.

Just leave these 2 lines as they were.